src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -137,25 +191,20 @@ export class SavedObjectsClient {
     };
   }
 
-  async get(type, id) {
-    const response = await this._withKibanaIndex('get', {
-      type,
-      id,
-    });
-
-    return {
-      id: response._id,
-      type: response._type,
-      version: response._version,
-      attributes: response._source
-    };
-  }
-
-  async update(type, id, attributes, options = {}) {
+  /**
+   * Updates an object
+   *
+   * @param {string} type
+   * @param {string} id
+   * @param {object} options
+   * @param {integer} options.version - ensures version matches that of persisted object
+   * @returns {promise}
+   */
+  async update(type, id, attributes, { version } = {}) {
```

Comment:
`{ version } = {}` is one of many different ways the options are read in this file, I'd love if they were all done the same way, preferably like so:

```js
async method(/* ... */, options = {}) {
  const {
    version: false
  } = options;

  // ...
}
```